api/com/dynatrace/oneagent/sdk/api/infos/TraceContextInfo.java

```diff
@@ -0,0 +1,38 @@
+package com.dynatrace.oneagent.sdk.api.infos;
+
+/**
+ * Provides information about the current PurePath node using the TraceContext
+ * (Trace-Id, Span-Id) model as defined in https://www.w3.org/TR/trace-context.
+ * The Span-Id represents the currently active PurePath node.
+ */
+public interface TraceContextInfo {
```

Comment:
We should add a caveat to use the header injectors instead of this class whenever possible. What is the recommended use case of this class? Loggging? Should we instead expose a single String getter returning a string in a format we can handle? (e.g. maybe `TRACEID-SPANID`)

Reply:
Those are different use cases. `TraceContextInfo` is meant for logging.
The reason that this PR contains trace header injector code APIs as well, is that it had never been synced here so far. Apart from that, this is unrelated.

Reply:
I suspected as much. Then, why do we need to have separate getters for trace and span ID? Do we expect that everybody wants to format it differently? If so, that would make ingesting the logs later more difficult. If not, why don't we just return a single string instead of this info object?

Reply:
That way we keep flexibility. Prescribing a fixed ingest format can turn into a legacy soon.

Reply:
OK, so we want to keep the format flexible and do not prescribe any fixed format? Because if we do, and we change it, somebody has to update code anyway, either us, if we return a single string, or the customer, if we have them format the final string themselves.

Reply:
I see how the previous comment was confusing. Prescribing in that context meant defining (in this spec SDK) how other SDKs would have to format a string.
Sure, we provide an ingest format to our customers. Should this change - yes then code must be adapted - but now a whole slew of SDK releases will be necessary.